lib/worldwide/address.rb

```diff
@@ -155,6 +155,18 @@ def concatenated_address2
       concatenate_fields(additional_fields)
     end
 
+    def parse_address1
+      additional_fields = region.additional_address_fields["address1"] || []
+      split_fields_arr = address1&.split(RESERVED_DELIMITER) || []
```

Comment:
Looking at the definition for RESERVED_DELIMITER above, it's not clear in most browsers, IDEs and text editors that we're using a non-breaking space as opposed to a regular space.

<img width="217" alt="image" src="https://github.com/Shopify/worldwide/assets/33674553/243fe7e1-e1ee-4f44-8d72-ddd99ef3d533">

Can we make that clearer by setting the constant to
```ruby
RESERVED_DELIMITER = "\u00A0" # non-breaking space
```